apps/docs/docs/react/install.mdx

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
`apps/docs/docs/react/install.mdx` double "docs" in path does not sound nice. May be rename the project to `site`?

Reply:
`docsite` ?